.env.dist

```diff
@@ -1,9 +1,8 @@
+##### All variables described in envs.md
 # App configuration
 #############################
 ## Required variables
 
-#### Access to admin panel (route: {app_domain}/admin)
-##### Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
 SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
```

Comment:
We need to remove it. To be sure that the password will not be used by default.

Reply:
As I know removed in separate PR